nginx.conf

```diff
@@ -89,6 +89,9 @@ server {
     location /docs/human-interface-guidelines/ {
         return 301 /docs/human-interface-guidelines;
     }
+    location /loki {
```

Comment:
This would redirect `/loki-something/something` as well. Use `location = /loki` to exact match the URL.